rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/connection/auth/UserTokenRpcService.java

```diff
@@ -17,7 +17,7 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarsource.sonarlint.core.rpc.protocol.backend.usertoken;
+package org.sonarsource.sonarlint.core.rpc.protocol.backend.connection.auth;
```

Comment:
This will lead to a breaking API change. Not a big deal, but as I suggested in a message earlier, I think we should document those API changes somewhere.